Alloy/Renderers/BoundingBoxesRenderer.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  BoundingBoxesRenderer.swift
+//  Alloy
+//
+//  Created by Eugene Bokhan on 22/04/2019.
+//
+
+import Metal
+import simd
+
+@available(iOS 11.3, tvOS 11.3, macOS 10.13, *)
+final public class BoundingBoxesRenderer {
+
+    public enum Errors: Error {
+        case wrongRenderTargetTextureUsage
+        case missingRenderTarget
+    }
+
+    // MARK: - Properties
+
+    /// Rectrangles in a normalized coodrinate system describing bounding boxes.
+    public var normalizedRects: [CGRect] = []
+    /// Rrefered fill color of the bounding boxes. Red is default.
```

Comment:
that's not the fill color

Reply:
I'll change it to `... border color of ...`.